docs/source/data/data_catalog.md

```diff
@@ -111,270 +66,35 @@ cars:
-  credentials: usda_credentials
-  params:
-    key: SOME_TOKEN
-    format: JSON
-    commodity_desc: CORN
-    statisticcat_des: YIELD
-    agg_level_desc: STATE
-    year: 2000
-```
-
-Note that `usda_credientials` will be passed as the `auth` argument in the `requests` library. Specify the username and password as a list in your `credentials.yml` file as follows:
-
-```yaml
-usda_credentials:
-  - username
-  - password
-```
-
-
-### Example 15: Loads data from Minio (S3 API Compatible Storage)
-
-
-```yaml
-test:
-  type: pandas.CSVDataSet
-  filepath: s3://your_bucket/test.csv # assume `test.csv` is uploaded to the Minio server.
-  credentials: dev_minio
-```
-In `credentials.yml`, define the `key`, `secret` and the `endpoint_url` as follows:
-
-```yaml
-dev_minio:
-  key: token
-  secret: key
-  client_kwargs:
-    endpoint_url : 'http://localhost:9000'
-```
-
-```{note}
-The easiest way to setup MinIO is to run a Docker image. After the following command, you can access the Minio server with `http://localhost:9000` and create a bucket and add files as if it is on S3.
-```
-
-`docker run -p 9000:9000 -e "MINIO_ACCESS_KEY=token" -e "MINIO_SECRET_KEY=key" minio/minio server /data`
-
-
-### Example 16: Loads a model saved as a pickle from Azure Blob Storage
-
-```yaml
-ml_model:
-  type: pickle.PickleDataSet
-  filepath: "abfs://models/ml_models.pickle"
-  versioned: True
-  credentials: dev_abs
-```
-In the `credentials.yml` file, define the `account_name` and `account_key`:
-
-```yaml
-dev_abs:
-  account_name: accountname
-  account_key: key
-```
-
-
-### Example 17: Loads a CSV file stored in a remote location through SSH
-
-```{note}
-This example requires [Paramiko](https://www.paramiko.org) to be installed (`pip install paramiko`).
-```
-```yaml
-cool_dataset:
-  type: pandas.CSVDataSet
-  filepath: "sftp:///path/to/remote_cluster/cool_data.csv"
-  credentials: cluster_credentials
-```
-All parameters required to establish the SFTP connection can be defined through `fs_args` or in the `credentials.yml` file as follows:
-
-```yaml
-cluster_credentials:
-  username: my_username
-  host: host_address
-  port: 22
-  password: password
-```
-The list of all available parameters is given in the [Paramiko documentation](https://docs.paramiko.org/en/2.4/api/client.html#paramiko.client.SSHClient.connect).
-
-## Create a Data Catalog YAML configuration file via CLI
-
-You can use the [`kedro catalog create` command to create a Data Catalog YAML configuration](../development/commands_reference.md#create-a-data-catalog-yaml-configuration-file).
-
-This creates a `<conf_root>/<env>/catalog_<pipeline_name>.yml` configuration file with `MemoryDataSet` datasets for each dataset in a registered pipeline if it is missing from the `DataCatalog`.
-
-```yaml
-# <conf_root>/<env>/catalog_<pipeline_name>.yml
-rockets:
-  type: MemoryDataSet
-scooters:
-  type: MemoryDataSet
+    open_args_load:
+      mode: "rb"
+      encoding: "utf-8"
 ```
```

Comment:
What about dataset-wide metadata? Should we include that in "additional settings"?

Reply:
I have no idea 🤷 If you want to include it, suggest me some text pls!

Reply:
@astrojuanlu @noklam @merelcht

Reply:
What do you mean with "dataset-wide metadata" @astrojuanlu ? Are you talking about the `metadata` attribute (https://github.com/kedro-org/kedro/pull/2537) that can be used by plugins, such as the preview stuff for viz?

Reply:
Sorry, exactly that, yes!

Reply:
Here's the ticket https://github.com/kedro-org/kedro/issues/2940